pyronearEngine/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.2.0a0'
```

Comment:
I think you might want to start with a `0.1.0.a0` rather than skipping to `0.2.0..` ;)